### PR TITLE
Lesson 13

### PR DESCRIPTION
**Pull request by netscro**
Lesson 13

### CODE REVIEW

File: src/com/javabasic/lesson13/HomeWork13.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.javabasic.lesson13;
+
+public class HomeWork13 {
+
+    public static void main(String[] args) {
+        // task 1
+        PersonGenerator persons = new PersonGenerator();
```
Comment (kolodgeeva) [inline src/com/javabasic/lesson13/HomeWork13.java:7]: The same mistake, name your variables more strictly. persons -> personGenerator, generator etc

File: src/com/javabasic/lesson13/HomeWork13.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        Person[] personsArray = persons.getRandomPersonsArray();
```
Comment (kolodgeeva) [inline src/com/javabasic/lesson13/HomeWork13.java:8]: A word array isn't needed if you use the plural form (persons). Persons or personArray. First variant is better

File: src/com/javabasic/lesson13/Human.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package com.javabasic.lesson13;
+
+public class Human {
+
+    private String name;
+    private String secondName;
+    private int age;
+    private int weight;
+    private int height;
+    private Person mother;
```
Comment (kolodgeeva) [inline src/com/javabasic/lesson13/Human.java:10]: Don't mix two tasks) Use Human type here.

**Comment by netscro** [comment]
Got it! Thanks a lot!

**Pull request merged**